# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dvbd/gcc43.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/dvbsched.cpp.orig
+++ ./src/dvbsched.cpp
@@ -29,6 +29,7 @@
 #include <iostream>
 #include <time.h>
 #include <getopt.h>
+#include <stdlib.h>
 #include <cerrno>
 #include <fstream>
 
--- ./src/utils.cpp.orig
+++ ./src/utils.cpp
@@ -23,6 +23,7 @@
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <unistd.h>
+#include <string.h>
 #include <iostream>
 #include <cctype>
 #include <cerrno>
--- ./src/dvbtuneparamscommon.cpp.orig
+++ ./src/dvbtuneparamscommon.cpp
@@ -4,6 +4,7 @@
 #include "dvbtuneparamscommon.h"
 #include "tune.h"
 
+#include <stdlib.h>
 #include <map>
 #include <iostream>
 
--- ./src/signals.cpp.orig
+++ ./src/signals.cpp
@@ -22,6 +22,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <stdlib.h>
 #include <unistd.h>
 #include <signal.h>
 #include <cerrno>

